indigo_app/views/tasks.py

```diff
@@ -214,6 +214,7 @@ def get_context_data(self, **kwargs):
 
         context['task_labels'] = TaskLabel.objects.all()
         context['place_workflows'] = self.place.workflows.all()
+        context['potential_assignees'] = User.objects.filter(editor__permitted_countries=self.country)
```

Comment:
Nice :) -- let's do the same on the form itself. Try accessing `form.assigned_to.queryset` here and see if you can steal from that queryset (once the queryset is filtered).